modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -1815,3 +1816,46 @@ translationAndAutosaveTest(
 		).toHaveText('Approved');
 	}
 );
+
+baseTest(
+	'Verify Journal Article ID field is not editable after assignment',
+	{
+		tag: '@LPD-39034',
+	},
+	async ({journalEditArticlePage, page, site}) => {
+		const systemSettingsPage = new SystemSettingsPage(page);
+		await systemSettingsPage.goToSystemSetting(
+			'Web Content',
+			'Administration'
+		);
+
+		const toggleSelector =
+			'label:has-text("Journal Article Force Autogenerate ID") >> input[type="checkbox"]';
+		const toggle = page.locator(toggleSelector);
+		if (await toggle.isChecked()) {
+			await toggle.click();
+		}
+
+		const articleTitle = 'Test Article';
+
+		await journalEditArticlePage.createArticleWithCustomArticleId(
+			page,
+			site,
+			articleTitle
+		);
+
+		await journalEditArticlePage.editArticle(articleTitle);
+
+		const articleIDField = page.locator(
+			'input[name="_com_liferay_journal_web_portlet_JournalPortlet_newArticleId"]'
+		);
+
+		await expect(articleIDField).toBeDisabled();
+
+		const autoArticleIdField = page.locator(
+			'input[name="_com_liferay_journal_web_portlet_JournalPortlet_autoArticleId"]'
+		);
+
+		await expect(autoArticleIdField).toBeDisabled();
+	}
```

Comment:
Clean process is needed because we are changing Web Content system settings to this test.

Wee need to wrap all the test case with `try { }` and clean after execution success or fail with `finally {}` and go to system settings and reset as default.

We can use the`Reset Default Values` dropdown or uncheck the checkbox manually.
![image](https://github.com/user-attachments/assets/9cdb54a3-00ff-462c-b545-1af51b18722b)